reboot_pending/reboot_pending.resource.ps1

```diff
@@ -0,0 +1,12 @@
+ if (Get-ChildItem "HKLM:\Software\Microsoft\Windows\CurrentVersion\Component Based Servicing\RebootPending" -EA Ignore) { return @{ RebootPending = $true } | ConvertTo-Json -Compress }
```

Comment:
Do we know if these reg paths are all publicly documented?

Reply:
Yes; I found them mentioned here:
https://learn.microsoft.com/en-us/mem/configmgr/core/servers/deploy/install/list-of-prerequisite-checks#pending-system-restart